### DISCUSSION TITLE
clone counter color

### DISCUSSION THREAD

**Discussion by ksdavidc** [ScratchAddons/ScratchAddons#6745] [Help]
Is there a way to adjust the color of the clone counter? It is hard to see in my color scheme.

**Comment by WorldLanguages** [comment]
It's not possible currently, I think there's been some recent discussion about this, but I couldn't find it

**Reply by WorldLanguages** [reply]
@mxmou Do you think solving this would be easy with customCssVariables in editor-dark-mode?

**Reply by WorldLanguages** [reply]
I was thinking we could switch automatically to white(?) if there's not enough contrast

**Reply by mxmou** [reply]
The simplest solution is to switch between a light and dark version of the color depending on the background. This would fix the text being impossible to read on certain background colors, but it wouldn't guarantee a 4.5:1 contrast ratio.

**Reply by WorldLanguages** [reply]
Sounds good to me

**Comment by DNin01** [comment]
I mentioned the idea of good text contrast against custom background colors in https://github.com/ScratchAddons/ScratchAddons/issues/5466#issuecomment-1712587198.

> Maybe it could choose a darker color only if the background was dark enough. If it was lighter, it'd have to use pure white for contrast. And a similar idea for black text on light backgrounds.

**Comment by ksdavidc** [comment]
see: https://github.com/ScratchAddons/ScratchAddons/issues/6638#issuecomment-1752130992
 includes this:

> - when you are talking about the text color, I am not sure that affects the color of the clone counter text. The clone counter color seems to controllable via .clone-container-container or possibly .clone-container or .clone-count.
> - This does not affect the color of the cat.svg,...

as well as other relevant (ha ha so I think) statements.

**Reply by WorldLanguages** [reply]
Looks like you figured out a way to change the image and text color through CSS :)

**Reply by WorldLanguages** [reply]
Of course, ideally everything should be customizable through the UI, but the ability to independently customize the look of any website is cool.
